Week3/solution.txt.txt

```diff
@@ -0,0 +1,216 @@
++-----+---------+-----------+-------+---------+
+| 106 | Krunal  | HR        | 21000 | Bardoli |
+| 107 | Radhe   | Account   | 25000 | sarbhon |
+| 108 | Rajesh  | IT        | 26500 | Navsari |
+| 109 | Dhruv   | Inventory | 20000 | Bardoli |
+| 110 | Grijesh | HR        | 23000 | surat   |
++-----+---------+-----------+-------+---------+
+'''
+
+#record insert using user input
+ins2="insert into employee values(?,?,?,?,?)"
+l=[ ] #create empty list
+for i in range(0,5):
+    n=int(input('Enter Employee Id: '))
+    name=input('Enter Employee Name: ')
+    dept=input('Enter Department Name: ')
+    basic=int(input('Enter Basic: '))
+    branch=input('Enter Branch: ')
+    t=[n,name,dept,basic,branch]
+    l.append(t)
+cur.executemany(ins2,l)
+con.commit()
+'''
++-----+---------+-----------+-------+--------+
+| eid |  ename  |   dept    | basic | branch |
++-----+---------+-----------+-------+--------+
+| 111 | John    | Inventory | 30000 | Vyara  |
+| 112 | Mohan   | Account   | 32000 | Delhi  |
+| 113 | Vikas   | HR        | 42000 | Pune   |
+| 114 | Vrijesh | IT        | 31000 | Mumbai |
+| 115 | Amit    | HR        | 41000 | Mumbai |
++-----+---------+-----------+-------+--------+
+'''
+
+3. Update records who are from "Surat" branch with increment in salary 10%.
+Ans:-
+
+import sqlite3 as sql
+con=sql.connect("c://sqlite3//ESM.db")
+cur=con.cursor()
+q1=  "update employee set \
+    basic=basic+basic*10/100\
+    where branch is 'surat'\
+    "
+cur.execute(q1)
+con.commit()
+
+4. Print All records.
+Ans:-
+
+import sqlite3 as sql
+con=sql.connect("c://sqlite3//ESM.db")
+cur=con.cursor()
+q2="select * from employee"
+cur.execute(q2)
+r=cur.fetchall()
+for i in r:
+    print(i)
+con.commit()
+
+5. Print records where dept is "HR" and "IT".
+Ans:-
+
+import sqlite3 as sql
+con=sql.connect("c://sqlite3//ESM.db")
+cur=con.cursor()
+q3="select * from employee where dept='HR' or dept='IT' "
+cur.execute(q3)
+r=cur.fetchall()
+for i in r:
+    print(i)
+con.commit()
+
+6. Print records in sorting order of department.
+Ans:-
+
+import sqlite3 as sql
+con=sql.connect("c://sqlite3//ESM.db")
+cur=con.cursor()
+
+#order by in desending order
+q4="select * from employee order by dept desc"
+cur.execute(q4)
+r=cur.fetchall()
+for i in r:
+    print(i)
+
+#order by in asending order
+query="select * from employee order by dept asc"
+#in sql all records are by default asending order
+cur.execute(query)
+r1=cur.fetchall()
+for i in r1:
+    print(i)
+con.commit()
+
+7. Print records where basic is >6000.
+Ans:-
+
+import sqlite3 as sql
+con=sql.connect("c://sqlite3//ESM.db")
+cur=con.cursor()
+q5="select * from employee where basic>6000"
+cur.execute(q5)
+r=cur.fetchall()
+for i in r:
+    print(i)
+con.commit()
+
+8. Print records whrere employee name second character is "r".
+Ans:-
+
+import sqlite3 as sql
+con=sql.connect("c://sqlite3//ESM.db")
+cur=con.cursor()
+q6="select * from employee where ename like '_r%'"

```

Comment:
why are you use '_' before R?

Reply:
bcoz we want r at 2nd position of name so i use  _ instead 1st character